### ISSUE TITLE
Keyboard use with mobile VO is inconsistant

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molecules-and-light#257]
For https://github.com/phetsims/QA/issues/461. Noticed on iPadOS 13.3 VO with a bluetooth keyboard.
First, on such sims there is no keyboard dialog. This is because it is on iPad, but it is possible to connect a keyboard.
Second, it seems to be very difficult and inconsistent getting a slider to work with keyboard commands. I've gotten it to work once by pressing up/down while focused on it, but often that doesn't work. I can't seem to determine the conditions it does work in.
If this is not a set up we plan to support, feel free to close. If it is, a more in-depth look at this set up is needed. I only did a quick look at this, so if we want to look at this seriously let me know and I can do more detailed testing on it.

**Comment by jessegreenberg** [comment]
Keyboard use with mobile VO is a little fuzzy.

In general we have not supported bluetooth keyboard with iOS because they don't work very well for a11y. When it apeared that we would looks iOS platform information, we tested and found in https://github.com/phetsims/QA/issues/428 that it worked OK. But then we were able to keep iOS platform info and so we continued to NOT support it with a keyboard (which is why the keyboard help dialog is hidden).

**Comment by jessegreenberg** [comment]
I just tried out keyboard navigation with a bluetooth keyboard with and without VO. OVerall I cannot tab navigate in basic HTML pages wth the bluetooth keyboard. Some text input is in the navigation order, but most are not. When VO is on sliders or other things arrow keys do not move them, but move the virtual cursor.

In https://github.com/phetsims/QA/issues/428 the balloon moved because it is specifically flagged to do as it is a custom widget. But native elements will generally not work.

@KatieWoe because of this we will not be supporting bluetooth keyboard on iOS or iPadOS, they just don't work well enough for the purposes of a11y. They seem to really only facilitate easier typing.
